packages/config/src/ConfigAutoStore.ts

```diff
@@ -106,7 +106,7 @@ export class ConfigAutoStore {
 
             if (authHandlerClass instanceof AbstractAuthHandler) {
                 const { promptParams } = authHandlerClass.getAuthHandlerApi();
-                if (profile.tokenType === promptParams.defaultTokenType) {
+                if (profile.tokenType.startsWith(promptParams.defaultTokenType)) {
```

Comment:
Curious why we are replacing the equality check with `startsWith`? It makes sense for token type `apimlAuthenticationToken`, but for other token types like JWT do we still want an equality check?

I guess we're a bit limited with our options here - as we can't redefine the array of token types as regexes (`^jwtToken$`, `^apimlAuthenticationToken.*`, etc) without a breaking change.